portfolio/src/main/webapp/scripts/utilities.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Ensures that the stated arguments are present. 
+ * @param {...(NodeList|HTMLCollection|HTML element|string|Array)} args
+ *    Arguments to be checked.
+ */
+function ensureNonNull(... args) {
+  /**
+   * Indicates error type.
+   * @enum {string}
+   */
+  const ErrorType = {
```

Comment:
If you want to define an enum, it should not be mutable.

You may want `Object.freeze(...)`

Reply:
Thanks for the suggestion! (: I didn't actually know how enums work in js. 😅